src/lib/components/globalhealthbar/GlobalHealthBar.component.js

```diff
@@ -0,0 +1,139 @@
+//@flow
+import React from 'react';
+import VegaChart from '../vegachart/VegaChart.component.js';
+
+type Props = {
+  id: string,
+  alertSegments: {
+    severity: 'critical' | 'warning' | 'none',
+    startSegment: string,
+    endSegment: string,
+    description: string,
+  }[],
+  start: { date: number, month: number, year: number },
```

Comment:
Date format taken as an input should be uniformed between startSegement/endSegment and start/end likely use a JS Date object or an ISO string

Reply:
Yes. I changed to ISO string.  
And then format it to this { date: number, month: number, year: number } which is required by the vega-lite library.